dependent/pontifical-gregorian-university-fr.csl

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" initialize-with-hyphen="true" page-range-format="expanded" demote-non-dropping-particle="sort-only" name-delimiter=" &#8211; " names-delimiter=" &#8211; " delimiter-precedes-last="always" et-al-min="4" et-al-use-first="1" name-form="long" initialize-with=". " sort-separator=", " default-locale="fr-FR">
```

Comment:
@rmzelle -- do you see any problems with the formatting attributes (like name delimiters etc.) here for a dependent style? Do you know, btw., if they would override attributes in a parent style (I assume not)?